dallinger/utils.py

```diff
@@ -996,3 +997,30 @@ def classproperty(func):
         func = classmethod(func)
 
     return ClassPropertyDescriptor(func)
+
+
+def median_time_spent_in_hours(participants: list[Participant]) -> float:
+    """
+    Parameters:
+        participants: list[Participant]
+            A list of participants to calculate the median time spent in hours for.
+
+    Returns:
+        float
+            The median time spent in hours by participants in a list of participants.
+    """
+    from statistics import median
+
+    if not participants:
+        return 0
```

Comment:
I think you should return `None` if there are no participants. Or alternatively, raise a ValueError

Reply:
Returning `None` now